src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs

```diff
@@ -199,37 +195,18 @@ public override void Open()
                     break;
             }
 
-            if ((flags & SQLITE_OPEN_URI) == 0
-                && !filename.Equals(":memory:", StringComparison.OrdinalIgnoreCase)
-                && !Path.IsPathRooted(filename))
-            {
-                filename = Path.GetFullPath(Path.Combine(BaseDirectory, filename));
-            }
-
-            var rc = NativeMethods.sqlite3_open_v2(filename, out _db, flags, vfs: null);
-            MarshalEx.ThrowExceptionForRC(rc, _db);
+            var rc = raw.sqlite3_open_v2(filename, out _db, flags, vfs: null);
+            SqliteException.ThrowExceptionForRC(rc, _db);
 
             SetState(ConnectionState.Open);
         }
 
-#if !NET451
-        private static string BaseDirectory
-            => Environment.GetEnvironmentVariable("ADONET_DATA_DIR")
-                ?? ApplicationDataHelper.LocalFolderPath
-                ?? AppContext.BaseDirectory;
-#else
-        private static string BaseDirectory
-            => AppDomain.CurrentDomain.GetData("DataDirectory") as string
```

Comment:
Nvm. Just realized you already opened https://github.com/ericsink/SQLitePCL.raw/issues/139. In the meantime, let's ensure we don't break this.